### DISCUSSION TITLE
Removed whitespace in templates

### DISCUSSION THREAD

**Discussion by triceo** [jreleaser/jreleaser#1423] [Q&A] [answered]
I have a JReleaser config like so:

            <release>
              <github>
                ...
                <changelog>
                  <formatted>ALWAYS</formatted>
                  <preset>conventional-commits</preset>
                  <contentTemplate>.github/workflows/release-changelog-template.md</contentTemplate>
                ...

The template has Markdown code like this:

    ```xml
        <dependency>
          <groupId>a</groupId>
          <artifactId>b</artifactId>
          <version>{{versionNumber}}</version>
        </dependency>
    ```

Yet in the resulting Release page on Github, the content always loses the whitespace:

    <dependency>
    <groupId>a</groupId>
    <artifactId>b</artifactId>
    <version>{{versionNumber}}</version>
    </dependency>

I have double-checked and the Github content already is like this. So it must be either JRelease or Mustache that removes my whitespace. What can I do to stop this?

**Comment by aalmiray** [comment, marked as answer]
This might be related to https://github.com/jreleaser/jreleaser/issues/1142 which has yet to be solved.
